bin/lint-migrations-file/src/change_set/strict.clj

```diff
@@ -22,10 +22,17 @@
    ;; different DBMSes
    :sql-changes-for-different-
    (s/and
-    (s/+ :change.strict/dbms-qualified-sql-change)
+    (s/+ (s/alt :sql-change :change.strict/dbms-qualified-sql-change
+                :sqlFile-change :change.strict/dbms-qualified-sqlFile-change))
```

Comment:
The changes to the spec in these files are just to ensure that the `lint-migrations-file` allows `sqlFile` changes to be split between different DB types, just like they can for `sql` changes